scripts/travis_install.sh

```diff
@@ -28,7 +27,7 @@ conda update -q conda
 conda info -a # for debugging
 
 conda remove --name test --all || true
-conda create -q -n test --quiet -c conda-forge notebook nodejs pytest python=$PYTHON
+conda create -q -n test --quiet -c conda-forge notebook pytest python=$PYTHON
```

Comment:
Why use the system nodejs, just curious? For speed (since it's already installed)?